### ISSUE TITLE
option to make Find case-sensitive

### ISSUE THREAD

**Issue by HotFusionMan** [warpdotdev/Warp#2552]
### Describe the solution you'd like?

```text
I would like to be able to specify that I want the Find functionality to be case-sensitive. I imagine there would be a setting added to the Features section of the preferences for this option.
```

### Is your feature request related to a problem? Please describe.

```text
Sometimes I want to find an occurrence of a text string that may appear very frequently in the large amount of console output text to be searched, but I only want to see the occurrences with specific capitalization/case.
```

### How important is this feature to you?

1 (Not too important)

### Warp Internal (ignore) - linear-label:770f6576-d6c0-4e4f-a259-fc64b5156087

None

**Comment by vorporeal** [comment]
We actually support this already!  You can toggle case-sensitive search on using this button within the find bar:

![image](https://user-images.githubusercontent.com/211194/214962557-2984a4eb-9d4d-4a8b-b757-a384335f7800.png)

**Comment by HotFusionMan** [comment]
Boo, I think I've even used that before! Can I request that the Find bar provide more visual contrast in light mode instead of my original request?

**Comment by vorporeal** [comment]
Sure!  When you say "in light mode", which specific theme are you using?  The "Light" theme?

**Comment by HotFusionMan** [comment]
Yes, the Light theme. Sorry to be imprecise, "dark mode" is talked about so much, I didn't even think about the terminology.
